internal/gensupport/resumable.go

```diff
@@ -135,7 +135,19 @@ func (rx *ResumableUpload) transferChunk(ctx context.Context) (*http.Response, e
 		return nil, err
 	}
 
-	res, err := rx.doUploadRequest(ctx, chunk, off, int64(size), done)
+	// rCtx is derived from a context with a defined transferTimeout with non-zero value.
+	// If a particular request exceeds this transfer time for getting response, the rCtx deadline will be exceeded,
+	// triggering a retry of the request.
+	var rCtx context.Context
```

Comment:
This is being defined and used at [this location](https://github.com/googleapis/google-api-go-client/blob/main/internal/gensupport/send.go#L89). Moving this timeout won't help.